src/sentry/static/sentry/app/components/group/inboxBadges/inboxReason.tsx

```diff
@@ -23,57 +23,83 @@ type Props = {
 function InboxReason({inbox, fontSize = 'sm'}: Props) {
   const {reason, reason_details, date_added: dateAdded} = inbox;
 
+  const getCountText = (count: number) =>
+    count > 1000 ? `More than ${Math.round(count / 1000)}k` : `${count}`;
+
   function getReasonDetails(): {
     tagType: React.ComponentProps<typeof Tag>['type'];
     reasonBadgeText: string;
     tooltipText?: string;
+    tooltipDescription?: string;
   } {
     switch (reason) {
       case GroupInboxReason.UNIGNORED:
         return {
           tagType: 'default',
           reasonBadgeText: t('Unignored'),
-          tooltipText: t('%(count)s events within %(window)s', {
-            count: reason_details?.count || 0,
-            window: moment.duration(reason_details?.window || 0, 'minutes').humanize(),
+          tooltipText:
+            dateAdded &&
+            t('Unignored %(relative)s', {
+              relative: getRelativeDate(dateAdded, 'ago', true),
+            }),
+          tooltipDescription: t('%(count)s events in %(window)s', {
+            count: getCountText(reason_details?.count || 0),
+            window: getDuration((reason_details?.window || 0) * 60, 0, true),
           }),
         };
       case GroupInboxReason.REGRESSION:
         return {
           tagType: 'error',
           reasonBadgeText: t('Regression'),
-          tooltipText: t('Issue was resolved'),
+          tooltipText:
+            dateAdded &&
```

Comment:
hmmm if dateAdded is undefined you end up getting an empty tooltip

![image](https://user-images.githubusercontent.com/9372512/102949349-17cc9a00-4496-11eb-9a84-fb7026c50b80.png)

you may have to add some additional logic to not include the tooltip if you get an undefined/falsey object

Reply:
You're right, if there is no date added there is no point to this tooltip